src/Common/ninjam/client/ClientMessages.cpp

```diff
@@ -387,9 +387,7 @@ ClientToServerChatMessage ClientToServerChatMessage::from(QIODevice *device, qui
 
     QByteArray byteArray(payload, Qt::Uninitialized);
 
-    int readed = stream.readRawData(byteArray.data(), payload);
-
-    Q_ASSERT(readed == payload);
+    Q_ASSERT(stream.readRawData(byteArray.data(), payload) == payload);
```

Comment:
@pljones , can you check if these changes will work when compiling for RELEASE?

From Qt docs:

```Q_ASSERT_X is useful for testing pre- and post-conditions during development. It does nothing if QT_NO_DEBUG was defined during compilation.
```

Reply:
Define "work".  The result of the evaluation is assigned to an usused variable.  The Q_ASSERT(xxx) argument simply doesn't appear in the source for RELEASE.

If _side effects_ are required, then they should be commented.  I don't propose to check that.